src/components/TopBar/TopBarStyles.js

```diff
@@ -0,0 +1,59 @@
+import styled from 'styled-components';
+
+const bottomLineHeight = 1;
+const topBarHeight = 56;
+
+export const StyledTopBarWrapper = styled.nav`
+  position: absolute;
+  top: 0;
+  right: 0;
+  width: 100%;
+  height: ${topBarHeight + bottomLineHeight}px;
+`;
+
+export const StyledTopBar = styled.div`
+  width: 100%;
+  height: ${topBarHeight}px;
+  padding: 16px;
+  background-color: #2a2a2a;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0px 4px 6px rgba(0, 0, 0, 0.12), 0px 2px 4px rgba(0, 0, 0, 0.14),
+    0px 1px 2px rgba(0, 0, 0, 0.2);
+`;
+
+export const StyledHeader = styled.header`
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 19px;
+  padding-left: 53px;
+  line-height: 22px;
+  color: #3c78c8;
+`;
+
+export const StyledControls = styled.div`
```

Comment:
[I think it should be `StyledNav = styled.nav` :)](https://www.semrush.com/blog/semantic-html5-guide/)

![image](https://user-images.githubusercontent.com/6686679/83453947-0d986f00-a464-11ea-9a78-6f5c7a5c636a.png)